lib/core/rancher/step.sh

```diff
@@ -10,14 +10,31 @@ function ih::setup::core.rancher::help() {
 # Otherwise return 1.
 function ih::setup::core.rancher::test() {
 
+    # Check if Rancher was installed manually
+    brew list rancher > /dev/null 2>&1
+    RANCHERINSTALLED=$?
+
+    if [ $RANCHERINSTALLED -eq 0 ] 
+    then
+        ih::log::debug "Rancher Desktop was installed manually and should be uninstalled"
+        return 1
+    fi
+
 
     # Check for PLIST FILE
     PLISTFILE="$HOME/Library/Preferences/io.rancherdesktop.profile.defaults.plist"
     if [ -f "$PLISTFILE" ]; then
-        return 0
+
+        # If this file exists, let's check kubernetes is not enabled
+        RESULT=$(rdctl list-settings | jq  '.kubernetes.enabled')
```

Comment:
I'm not sure what will happen if Rancher Desktop isn't running here, but this probably won't work.

Reply:
I totally agree with you. I change the validation in order to see if our cask is listed or not. 
I let the file validation just like we had it.